### PR TITLE
editorial: mark currencySystem at risk

### PR DESCRIPTION
**Pull request by marcoscaceres**
***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/browser-payment-api/feat_at_risk.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/browser-payment-api/bf19415...967d29b.html)

### CODE REVIEW

**Review by adrianhopebailie — COMMENTED** [review]
Chairs approve marking this at risk per consensus at F2F meeting in Chicago. Suggest another editor do editorial review.

File: index.html [hunk 1]
```diff
@@ -99,6 +99,30 @@
...
+          Features at risk
+        </h2>
+        <p>
+          As this specification enters the Candidate Recommendation phase of
+          the W3C standardization process, the working group has identified the
+          following feature(s) as being "<dfn>at risk</dfn>" of being removed
+          from the specification. The working group seeks input from
+          implementers, developers, and the general public on whether these
+          features should remain in the specification. If no compelling use
+          cases are received, these features will be removed from the
```
Comment (domenic) [inline index.html]: What if we receive compelling use cases, but still nobody implements?

Comment (ianbjacobs) [reply index.html]: Seems ok to add "or if there is insufficient implementation experience"

Ian

Comment (marcoscaceres) [reply index.html]: > or if there is insufficient implementation experience

It's not about experience, it's about implementer interest. Updated text to reflect that.

**Review by adrianba — APPROVED** [review]

**Pull request merged**